dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapConfigManager.java

```diff
@@ -634,10 +640,13 @@ private void setFilters(Map<String, Object> configProps) throws WIMSystemExcepti
 
                 // Set the login property.
                 // Remove the default uid login Property if userFilter is configured.
```

Comment:
This comment still valid?

Reply:
Kind of, but I will reword it to make it more clear.